### PR TITLE
Make dbaccessors for users_by_location

### PR DESCRIPTION
**Pull request by esoergel**
@nickpell

### CODE REVIEW

File: corehq/apps/locations/dbaccessors.py [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
         'locations/users_by_location_id',
         startkey=[location_id],
         endkey=[location_id, {}],
-        include_docs=True
-    ).all()
+        include_docs=include_docs,
+    )
+
+
+def get_users_by_location_id(location_id, wrap=True):
```
Comment (dannyroberts) [inline corehq/apps/locations/dbaccessors.py]: I've been moving away from using a wrap parameter in the function I've been writing, in favor of just writing two different functions (get_user_json vs get_user), just to stick to having each public function have a consistent return type.

Comment (dannyroberts) [inline corehq/apps/locations/dbaccessors.py]: I'm open to being told that's not a worthwhile tradeoff; I've just seen `my_ambiguous_return_type_function(wrap=self.wrap)` etc. too many times—what's the return type there!? It encourages passing these options through that are intended to be passed directly to a couchdbkit call, which leads to pretty bad interfaces that are hard to refactor

Comment (esoergel) [inline corehq/apps/locations/dbaccessors.py]: It sounds reasonable to me, I was thinking about that when I made this - you can see I used 3 different functions which each wrap the same private function.  I'll go ahead and do the same for this parameter.

Comment (esoergel) [inline corehq/apps/locations/dbaccessors.py]: https://github.com/dimagi/commcare-hq/pull/7176

**Pull request merged**